File: src/test/java/SecuredMethodJavaTest.java
import com.assertj.SecuredMethodJava;
import org.junit.Test;

import static junit.framework.TestCase.assertNull;
import static org.junit.Assert.assertNotNull;

public class SecuredMethodJavaTest {

    @Test
    public void theUnlockedMethodShouldReturnAValue() {
        SecuredMethodJava service = new SecuredMethodJava();

        String value = service.unlockedMethod();

        assertNotNull(value);
    }

    @Test
    public void theLockedMethodShouldReturnNull() {
        SecuredMethodJava service = new SecuredMethodJava();

        String value = service.lockedMethod();

        assertNull(value);
    }
}

File: src/test/kotlin/SecuredMethodJavaTest.kt
import org.junit.Assert.assertNotNull
import org.junit.Assert.assertNull
import org.junit.Test

class SecuredMethodJavaTest {

    @Test
    fun `The unlocked method should return a value`() {
        val service = SecuredMethodJava()

        val value = service.unlockedMethod()

        assertNotNull(value)
    }

    @Test
    fun `The locked method should return null`() {
        val service = SecuredMethodJava()

        val value = service.lockedMethod()

        assertNull(value)
    }
}